src/views/migration/constant.ts

```diff
@@ -0,0 +1,12 @@
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+export const MTV_OPERATOR = 'mtv-operator';
+
+export const MTV_ROUTE_NAME = 'virt';
```

Comment:
according to [1] the correct way is to look for a label "service: forklift-ui"
" The URL for the forklift-ui service that opens the login page for the MTV web console is displayed."
 
( if we are using the documentation we are safer when breaking changes are done ) 

[1] https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html/installing_and_using_the_migration_toolkit_for_virtualization/installing-the-operator#obtaining-console-url_mtv

Reply:
Instead [here](https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html/installing_and_using_the_migration_toolkit_for_virtualization/installing-the-operator#obtaining-console-url_mtv) the command to get the MTV web console URL is 
```$ oc get route virt -n openshift-mtv -o custom-columns=:.spec.host``` .
This means that I have to get the resource `route` named `virt` in the namespace `openshift-mtv` and use the `spec.host` attribute.

Maybe these two methods are equivalent?